glib/src/enums.rs

```diff
@@ -631,6 +658,23 @@ impl Clone for FlagsClass {
     }
 }
 
+#[derive(Debug)]
+pub struct ParseFlagsError(String);
+
+impl std::error::Error for ParseFlagsError {}
+
+impl fmt::Display for ParseFlagsError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "Unknown flag: '{}'", self.0)
+    }
+}
+
+impl From<ParseFlagsError> for String {
```

Comment:
Do we need to have a public constructor of the parse error type?

Reply:
This is the other way around :slightly_smiling_face:  I just wanted a way to get at the underlying string, I changed it to an explicit getter method